docs/4.1/components/badge.md

```diff
@@ -80,3 +80,23 @@ Using the contextual `.badge-*` classes on an `<a>` element quickly provide _act
 <a href="#" class="badge badge-{{ color.name }}">{{ color.name | capitalize }}</a>{% endfor %}
 {% endcapture %}
 {% include example.html content=example %}
+
+## Close icon
+
+Create "dismissable" badges by adding the `.close` icon within a badge. Note that `<button>` and `<a>` elements are not supported _within_ another `<a>`. No JavaScript is included in Bootstrap to do the dismissing for you, so you'll have to write your own to implement any desired behaviors.
+
+{% capture example %}
+<span class="badge badge-primary">
+  Dismissable badge
+  <button type="button" class="close" aria-label="Dismiss">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</span>
+<a href="#" class="badge badge-secondary">
+  Dismissable link badge
+  <span role="button" class="close" aria-label="Dismiss">
```

Comment:
@mdo Thanks for pulling this one in. The error on the build is "An element with the attribute "role=button" must not appear as a descendant of the "a" element."

Since we need to bring the semantics of a button to the span here, can't we skip this example and leave just the one that has a button?

Reply:
@andresgalante I changed to the `<span>` to try to get around the same error for having a `<button>` in a `<a>`. Either way, it errors out for us.